frontend/src/main/web/lib/components/glossary/DataTable.jsx

```diff
@@ -140,7 +141,7 @@ var DataTable = React.createClass({
       }
     }
     if(StringUtils.isEmptyOrNull(title)) {
-      title = "No information available";
+      title = Actions.NO_INFO_MESSAGE;
```

Comment:
I would not think to look in `Actions` for a message. Is it the right place for that sort of thing?